base/applications/sndvol32/misc.c

```diff
@@ -158,6 +158,116 @@ CloseAppConfig(VOID)
     }
 }
 
+BOOL
+LoadXYCoordWnd(IN PREFERENCES_CONTEXT PrefContext)
+{
+    HKEY hKey;
+    LONG lResult;
+    TCHAR DeviceMixerSettings[356];
```

Comment:
AppRegSettings is 64 chars, DeviceName is 128, so something a little smaller would be sufficient (maybe 256, which is actually a power of two).